application/src/main/java/net/corda/v5/application/messaging/FlowMessaging.java

```diff
@@ -125,6 +125,25 @@ public interface FlowMessaging {
     FlowSession initiateFlow(@NotNull MemberX500Name x500Name, @NotNull FlowContextPropertiesBuilder flowContextPropertiesBuilder);
 
     /**
+     * Invoke an interop facade.
+     *
+     * This API is work in progress and is expected to change as interop functionality matures.
+     *
+     * @param memberName The MemberX500name of the alter-ego.
+     * @param facadeName The name of the facade to invoke.
+     * @param methodName The name of the function within the facade to invoke.
+     * @param payload The payload for the facade call.
+     * @return The response string from the facade.
+     */
+    @Suspendable
+    @NotNull
+    String callFacade(@NotNull MemberX500Name memberName, @NotNull String facadeName, @NotNull String methodName, @NotNull String payload);
```

Comment:
shouldn't the name of the function / API reflect InterOp ?
IOW `callInterOpFacade`

(There are no other Facades except InterOp ones)

Reply:
The final naming hasn't been solidify yet.

Reply:
I second this, highly unlikely that this form of the API is final.